### DISCUSSION TITLE
Does renovate support Gerrit?

### DISCUSSION THREAD

**Discussion by saiprashanthramasetti** [renovatebot/renovate#11433] [Request Help] [answered]
Hi,
It didn't look like renovate supports Gerrit code review tool. But has anyone tried out of the box implementation for using renovate via Gerrit. On the back code is pushed to Bitbucket after Gerrit review and approvals.

**Comment by viceice** [comment]
Nope, Gerrit isn't supported

**Comment by viceice** [comment, marked as answer]
Gerrit is now supported!

- https://github.com/renovatebot/renovate/releases/tag/37.112.0
- https://github.com/renovatebot/renovate/pull/18961
